demos/object_detection_demo/cpp/main.cpp

```diff
@@ -343,8 +350,19 @@ int main(int argc, char *argv[]) {
                     }
                 }
 
+                frameSize = curr_frame.size();
+                fps = cap->fps();
+
                 frameNum = pipeline.submitData(ImageInputData(curr_frame),
-                    std::make_shared<ImageMetaData>(curr_frame, startTime));
+                                               std::make_shared<ImageMetaData>(curr_frame, startTime));
```

Comment:
Why the alignment is changed? I believe it's better to have the one which was there: it doesn't require realigning if the position of `(` on previous line changes.

Reply:
changed back. That was done by autoformatter